doc/impl.md

```diff
@@ -148,8 +148,8 @@ to keep the implementation terse, but should not hinder normal usage:
 * The garbage collector recurses on the `CAR` of objects thus deeply nested
   `CAR`s may overflow the C stack — an object's `CDR` is looped on and will not
   overflow the stack
-* The storage of an object's type and GC mark assumes a little-endian system and
-  will not work correctly on systems of other endianness
+* The storage of an object's type and GC mark assumes a little-endian or
+  big-endian system and will not work correctly on systems of other endianness
```

Comment:
"other endianness"?

Reply:
I have heard that there is mixed-endian (not bi-endian), but I don't even know where it is actually used. So I'm not sure if the issue is completely solved.

Reply:
Well, mixed-endian is rather academic
https://en.wikipedia.org/wiki/Endianness#Middle-endian

Reply:
Well, mixed-endian is rather academic
https://en.wikipedia.org/wiki/Endianness#Middle-endian

Reply:
I have a new phobia: the obsessive feeling that there is a third byte order somewhere.

Reply:
FYI I made a big endian PowerPC port http://aminet.net/package/dev/lang/fe

Reply:
From personal experience, I realized that it is very inconvenient to check every time you need `#define FE_BIGENDIAN`. So, I made an automatic endian check. If you consider it necessary, you can update your port.